### ISSUE TITLE
Adding complexity / number of parameters to the table

### ISSUE THREAD

**Issue by ialhashim** [Cadene/pretrained-models.pytorch#101]
Thanks for this valuable resource! if its possible could you also add the complexity or number of parameters in the table in the README.md

Also, have you looked at ShuffleNet v2? Here is a really good implementation of it https://github.com/ericsun99/Shufflenet-v2-Pytorch

**Comment by Cadene** [comment]
I will certainly add it to the table. Please have a look at this paper for now:
https://arxiv.org/pdf/1810.00736.pdf

Thanks for ShuffleNetV2 :) I will have a look
